src/modules/access/common/WalletInterface.ts

```
@@ -0,0 +1,160 @@
+    this.publicKey = null;
+    this.isPubOnly = true;
+    this.isAddress = false;
+
+    if (!isPub) {
+      const _privateKey = Buffer.isBuffer(key) ? key : getBufferFromHex(key);
+      try {
+        if (_privateKey.length !== 32 || !isValidPrivate(_privateKey)) {
+          throw new Error('Private key does not satisfy the curve requirements (ie. it is invalid)');
+        }
+      } catch (e) {
+        throw e;
```

Biome

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)